### ISSUE TITLE
cleaning demographic variable text fields

### ISSUE THREAD

**Issue by martinamorris** [nmarquezuw/fatalencounters#6]
The race variable text needs to be cleaned.  I thought this had already been done in the previous cleaning/harmonization scripts.  So if those scripts are not being used now, we should incorporate the parts that are relevant to FE data.

From the shinyapp descriptive stats page:

. | Race | N
-- | -- | ---
1 |African-American/Black | 7637
2 | Asian/Pacific Islander | 521
3 | European American/White | 5
4 | European-American/White | 13363
5 | Hispanic/Latino | 4563
6 | HIspanic/Latino | 1
7 | Middle Eastern | 48
8 | Native American/Alaskan | 275
9 | Other Race | 9
10 | Race unspecified | 1376

* combine obvious categories; Middle Eastern -> Other; "Race unspecified" text changed to "Missing"

1	Female	2664
2	Male	25432
3	Transexual	1
4	Transgender	15
5	White	1
6		98

* combine Transexual into Transgender, White = NA, label the NAs as "Missing"

Age | N
-- | --
1 | (0,15] | 609
2 | (15,35] | 14854
3 | (35,65] | 10914
4 | (65,Inf] | 782
5 |   | 1052

Modify labels to be human friendly (e.g., 0,14; 15-34 ...);  add "Missing" label to table, but not to data (age needs to be a numeric variable).

**Comment by nmarquezuw** [comment]
Fixed with commit [9687566e1db22c1dd03d167d07d67a2cf10dd33e](https://github.com/nmarquezuw/fatalencounters/commit/9687566e1db22c1dd03d167d07d67a2cf10dd33e). I made the fixes on the server side only (app.R) so the data attached to the package still has these issues. I wanted to have a meeting with you on the best way to clean the data set first and how to document the changes before having them reflected in the package.
